.github/workflows/CI.yaml

```diff
@@ -26,14 +26,23 @@ jobs:
         os: [macOS-latest, ubuntu-latest, windows-latest]
         python-version: ["2.7", "3.5", "3.6", "3.7", "3.8", "3.9", "3.10"]
```

Comment:
python-version: ["3.7", "3.8", "3.9", "3.10"]

Reply:
We _can_ do NEP29 for alchemtest but to be honest, as long as it works I'm ok with just testing on supported platforms. 

If you want to follow NEP29, open an issue.